pkg/kubelet/container/runtime.go

```diff
@@ -157,6 +159,8 @@ type ImageService interface {
 	RemoveImage(image ImageSpec) error
 	// Returns Image statistics.
 	ImageStats() (*ImageStats, error)
+	// ImageFsInfo gets filesytem information of images.
+	ImageFsInfo() (cadvisorapiv2.FsInfo, error)
```

Comment:
Ideally this API would be `ImageFsInfo() (runtimeapi.FsInfo, error)`, and would replace both ImageStats, and ImageFsInfo here.  cAdvisor wouldnt be part of this api, but just an implementation detail.  I would much rather change the kubelet to use runtimeapi.FsInfo than use a cAdvisor API